spec/models/person/add_request_spec.rb

```diff
@@ -43,6 +43,23 @@
       expect(people).to match_array([admin, topper].collect(&:id))
     end
 
+    it 'contains people without primary group this primary group layer' do
+      admin = Fabricate(Group::TopLayer::TopAdmin.name, group: groups(:top_layer)).person
+      topper = Fabricate(Group::TopGroup::Member.name, group: groups(:top_group)).person
+      bottom = Fabricate(Group::BottomLayer::Leader.name, group: groups(:bottom_layer_one)).person
+      # second role in layer
+      Fabricate(Group::TopGroup::Member.name, group: groups(:top_group), person: bottom)
+      Person::AddRequest::Group.create!(
+        person: topper,
+        requester: bottom,
+        body: groups(:bottom_layer_one),
+        role_type: Group::BottomLayer::Member.sti_name)
+
+      requests = Person::AddRequest.for_layer(groups(:top_layer))
+
+      expect(requests.first.person).to eq(topper)
+    end
+
```

Comment:
Da braucht's sicher noch Tests für gelöschte Personen. Darum geht es ja in diesem Issue.